cookiecutter/{{cookiecutter.github_repo_name}}/.github/ISSUE_TEMPLATE/bug.yml

```
@@ -0,0 +1,66 @@
+# GitHub issue form. For more information see:
+# https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms
+
+name: Report something is not working properly 🐛
+description:
+  Use this if there is something that is not working properly.  If you are not
+  sure or you need help making something work, please ask a question instead.
+labels:
+  - "priority:❓"
+  - "type:bug"
+body:
+  - type: markdown
+    attributes:
+      value:
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Please tell us what happened that shouldn't have.
+      placeholder: What happened that shouldn't have.
+    validations:
+      required: true
+  - type: textarea
+    id: what-expected
+    attributes:
+      label: What did you expect instead?
+      description: Please tell us what did you expect to happen.
+      placeholder: What did you expect to happen.
+    validations:
+      required: true
+  - type: input
+    id: version
+    attributes:
+      label: Affected version(s)
+      description:
+        Please add a comma-separated list of the versions affected by this
+        issue.
+      placeholder: 'Example: v0.11.0, v0.12.0'
+  - type: dropdown
+    id: part
+    attributes:
+      label: Affected part(s)
+      description:
+        Which parts of the repo are affected by this issue? Select all that
+        apply.
+      multiple: true
+      options:
+        - I don't know (part:❓)
+        - Documentation (part:docs)
+        - Unit, integration and performance tests (part:tests)
+        - Build script, CI, dependencies, etc. (part:tooling)
+        # TODO(cookiecutter): Add other parts
+        # Please have in mind that that the part:xxx should be created as
+        # a issue label in the GitHub repository"
```

should be -> need to be, to make it sound less like a suggestion and more like a requirement.

```suggestion
        # Please have in mind that that the part:xxx labels need to
        # be created in the GitHub repository.